iceoryx_examples/singleprocess/README.md

```diff
@@ -14,6 +14,8 @@ The example can be started with
 ./build/iceoryx_examples/singleprocess/single_process
 ```
 
+<!-- @todo Replace this with asciinema recording before v1.0-->
+
```

Comment:
The description of the sender/receiver functions are out of sync.

Reply:
Fixed by @elBoberido already